src/main/scala/com/google/cloud/spark/bigquery/BigQueryUtil.scala

```diff
@@ -65,4 +66,7 @@ object BigQueryUtil {
   }
 
   def noneIfEmpty(s: String): Option[String] =  Option(s).filterNot(_.trim.isEmpty)
+
+  def convertAndThrow(error: BigQueryError) : Unit =
+    throw new BigQueryException(UNKNOWN_CODE, error.getMessage, error)
```

Comment:
Is it appropriate to always use UNKNOWN_CODE here?

Reply:
The code is modeled after [BigQueryException.translateAndThrow()](https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryException.java#L98). As [BigQueryError](https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryError.java) does not contain the code, I see no other option.